pkg/k8shandler/service.go

```diff
@@ -56,13 +56,14 @@ func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateServices() error
 		return fmt.Errorf("Failure creating service %v", err)
 	}
 
+	//legacy metrics service that likely can be rolled into the single service that goes through the proxy
 	annotations["service.alpha.openshift.io/serving-cert-secret-name"] = fmt.Sprintf("%s-%s", dpl.Name, "metrics")
 	err = createOrUpdateService(
 		fmt.Sprintf("%s-%s", dpl.Name, "metrics"),
 		dpl.Namespace,
 		dpl.Name,
-		"metrics",
-		60000,
+		"restapi",
```

Comment:
this doesn't look correct...

Reply:
This is correct because now we have a single proxy that guards elasticsearch.  This means the metrics service needs to go through the one and only proxy.  Ideally we should be able to remove this service, possibly moving the annotation to the rest service.

Reply:
shouldn't the port be `9200` though?

Reply:
reverted to listen on 60000 but target restapi